.textlintrc.json

```diff
@@ -303,6 +303,7 @@
         ["colour", "color"],
         ["config\\b", "configuration"],
         ["DarkSky", "Dark Sky"],
+        ["e.g.", "e.g.,"],
```

Comment:
```suggestion
        ["e\\.g\\.", "e.g.,"],
```

Reply:
It's a regular expression, so we need to escape the `.` to prevent it is being used in the expression as any character.

Add the `i.e.` in as well please 👍